src/nxdoc/nuxeo-add-ons/nuxeo-retention-management/nuxeo-retention-installation/nuxeo-retention-installation-standard.md

```diff
@@ -208,22 +209,10 @@ To configure the Retention addon with a dedicated S3 bucket with Object Lock, fi
 
 Records can also be stored in the same bucket as the rest of the content. This provides a simpler architecture, but restricts the possibility to adapt to more complex scenarios in the long run.
 
-[<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More info](#s3-one-bucket-configuration)
-
-{{> anchor 's3-one-bucket-configuration'}}
-#### Storing Records in the Same Bucket as Other Documents
-
-This configuration consists on using the same S3 bucket for records and other documents.
-
-In this case, you can use the Amazon S3 addon with the [default configuration]({{page page='amazon-s3-online-storage'}}) with the Retention addon.
-
-This configuration does NOT allow using the [Amazon S3 Object Lock](https://docs.aws.amazon.com/AmazonS3/latest/dev/object-lock.html) feature (no WORM storage option).
-
 {{#> callout type='warning'}}
-This configuration is only compliant with the binary manager `org.nuxeo.ecm.core.storage.sql.S3BinaryManager`.
+It is CRITICAL to store both regular and record blobs in different locations in your S3 bucket. Always use different bucket prefixes. Otherwise, this will result in a shared storage configuration that will prevent the [Orphaned Blobs GC]({{page page='garbage-collecting-orphaned-blobs'}}) from running efficiently.
```

Comment:
If it's possible to use different buckets, it should be rewritten.
You write it's critical but you state the consequence is just bad performance on blob gc ?
btw is this for immediate, full or both blob gc ?

Reply:
Also, when starting with such configuration don't we have WARN on start ?
if yes it could be interesting to mention this, so it's easy to check (or find this doc page)